### DISCUSSION TITLE
云规则建议

### DISCUSSION THREAD

**Discussion by LanFeng11** [dreamncn/Qianji_auto#147] [奇思妙想]
### 描述建议

建议整理云规则分类，名称，描述，建议加一个云规则的数据反馈

### 该功能带来的好处或者优点

```bash
目前云规则比较多，名称不规范，分类杂乱，描述不清楚，还有一些重复了的，低质量的，用户比较难找到合适的规则。有一些规则适配不完全，或者是失效了，如果用户能标注他的缺陷，并反馈数据，能有效提高规则的质量
```

### 这不是重复的 issue

- [X] 我已经搜索了[现有 issue](https://github.com/dreamncn/Qianji_auto/issues)，以确保该功能尚未被报告。

**Comment by dreamncn** [comment]
这个后期需要整理的，目前暂无计划。
